examples/14-Csound/3-DigitalInOut/_main.csd

```diff
@@ -2,9 +2,9 @@
 ; Digital In/Out - Audio through - Csound on Bela
 ; 	* Bernt I. Wærstad, Victor Lazzarini, Alex Hofmann
 ;
-; This example uses a switch to turn a LED and audio on and off. 
+; This example uses a switch to turn a LED and audio on and off.
 ;
-; - connect a switch to +5V on one side and other side to digital pin 0
+; - connect a switch to +3.3V on one side and other side to digital pin 0
 ; - connect a LED + a 10K Ohm resistor between GND and digital pin 1
```

Comment:
probably want a 470ohm resistor or similar here. I doubt you will get much brightness out of a 10k one